master/getting-started/kubernetes/index.md

```diff
@@ -2,82 +2,40 @@
+enforces high-level, intent-focused network policy.  Together, Calico and Kubernetes provide a secure,
+cloud-native platform that can scale your infrastructure to hundreds of thousands of workloads.
 
 ## Installing Calico for Kubernetes
 
-There are two main ways to install Calico on Kubernetes.
-
-#### [Hosted Install Guide]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/hosted)
-
-This method uses Kubernetes manifests to install Calico for Kubernetes, using Kubernetes.
-Calico manifests ensure the necessary components are installed and run on each node in the cluster.
-
-#### [Integration Guide]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/integration)
-
-This method walks through the necessary manual steps to integrate Calico with your own deployment scripts and tools.  Follow
-this guide if you're integrating Calico with your own configuration management tools.
-
-## Guides
-
-The following guides walk through launching a CoreOS based Kubernetes cluster on various providers, using a
-Calico self-hosted installation.
-
-#### [AWS]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/aws)
-
-This guide covers launching a Kubernetes cluster on AWS with Calico.
-
-#### [GCE]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/gce)
-
-This guide covers launching a Kubernetes cluster on GCE with Calico.
-
-#### [Vagrant and Container Linux by CoreOS ]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/vagrant/)
-
-This guide uses Vagrant and VirtualBox to locally run a Kubernetes
-cluster with Calico.
+There are a number of ways to install Calico and Kubernetes.  The [installation documentation](installation)
+includes links to a number of popular guides and installers which use Calico. It also
+includes information on installing Calico on a from-scratch Kubernetes cluster using either a self-hosted Kubernetes manifest,
+or by integrating Calico into your own configuration management scripts.
 
 ## Using Calico with Kubernetes
 
-The following tutorials walk through policy configurations for Calico in Kubernetes.
-
-Before following these tutorials, ensure you've used the above guides to launch
-a Kubernetes cluster with Calico installed.
-
-#### [Simple Policy](tutorials/simple-policy)
-
-The Simple Policy Demo shows how to use Calico to secure a simple two-tier application in kubernetes.
-
-#### [Stars Demo](tutorials/stars-policy/)
+Once you have a Kubernetes cluster with Calico installed, the following articles will help you
+get familiar with Calico and make the most of the features that Calico provides.
 
-The Stars Demo features a UI which actively shows blocked and allowed
-connections as policy is implemented.
+##### Tutorials
 
-#### [Advanced Policy Demo](tutorials/advanced-policy)
+**[Using NetworkPolicy](tutorials/simple-policy)**: this guide explains how to use Calico to secure a simple two-tier application
```

Comment:
NetworkPolicy -> 'Network Policy' or 'the NetworkPolicy API' or 'Kubernetes NetworkPolicy' or 'Kubernetes Network Policy'?